include/deal.II/fe/fe_values.h

```diff
@@ -2138,8 +2138,9 @@ class FEValuesBase : public Subscriptor
   virtual ~FEValuesBase() override;
 
 
-  /// @name ShapeAccess Access to shape function values. These fields are filled
-  /// by the finite element.
+  /// @name ShapeAccess Access to shape function values
```

Comment:
`ShapeAccess` should be removed for the same reason as https://github.com/dealii/dealii/issues/8192. Might imply that other parts of the docs need changing

Reply:
I don't feel strongly about this, but also these comments should be within a `/*` `*/` to be consistent with the rest of the docs.

Reply:
I can do this in a separate PR (there are many places)

Reply:
Agreed. Its not that important by comparison to what you've fixed here.